### ISSUE TITLE
flash Problem

### ISSUE THREAD

**Issue by Feuer001** [Edzelf/ESP32-Radio#125]
I have a flash problem.
When I try to flash ESP32 radio I have display problems, e.g. a white display or rainbow pixels. (pin config is ok)
If I flash the radio .bin file then everything is ok.
with which settings was the .bin compiled?
I have the following settings:
ESP32 DEV modules;
default;
DOUT;
80MHz;
4M (32Mb);
115200;
none;
(on Apple Mac)

**Comment by Edzelf** [comment]
The settings are okay.  Check if you have the latest version of the libraries and the ESP core.

**Comment by Feuer001** [comment]
Yes I have the last versions.
Adafruit gfx 1.2.2
Adafruit ST7735 1.0.8
The display works well, only with the lines I have rainbow pixel. (first the line is green then about 1 second later there are rainbow pixels, but only with the Accu line and the volume line.)

**Comment by Edzelf** [comment]
I used Adafruit gfx 1.2.3.

**Comment by Feuer001** [comment]
![esp](https://user-images.githubusercontent.com/39369090/40353599-2d3f6cf0-5db2-11e8-8ead-4a8bc7a64be6.jpeg)

with Adafruit gfx 1.2.3 the Same problem

**Comment by Feuer001** [comment]
If I stop the playback, then the display works fine.
When I start the playback then I have rainbow lines again.

**Comment by Edzelf** [comment]
Looks like the CS of the VS1053 interferes with the CS of the display.  Are you sure that they are separated and work correctly?
I saw other users with this problem, but I can not reproduce it.

**Comment by Feuer001** [comment]
The hardware must be ok, because the display works with the .bin file

**Comment by Edzelf** [comment]
Yes, that's a good argument.  It there a reason not to use the bin file?

**Comment by Feuer001** [comment]
yes, I am waiting for a 2.8 "(240 * 320) display ili9341 (for a larger font) or is there any work in progress?

Problem solved!!
I have pin 15 (tft_cs)
changed to pin 22, that's it!

Another question:
Can you still integrate Bluetooth to stream music from mobile phone to esp-radio?

**Comment by Feuer001** [comment]
Yes and that on 2x TTGO T1 ESP-32 V1.0.0 Rev1.
But I ordered more ESP32, let's see if they work.

I'm curious if I can do it with the display ...

**Comment by konektor001** [comment]
I had the same problem and changing the CS pin to 22 solved me to.
Thanks.
